app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/screens/details/DetailViewController.kt

```diff
@@ -63,6 +65,13 @@ class DetailViewController(bundle: Bundle) : Controller(bundle), KoinComponent,
       router.popCurrentController()
     }
 
+    refreshView.setOnRefreshListener {
+      launch {
+        refreshView.isRefreshing = false
+        viewModel.getPokemonDetail(pokemon.id)
+      }
+    }
```

Comment:
this is a presenter responsibility, yeah?

Reply:
yes i had update the code according to feature you can see this update in UPDATING architecture branch